### ISSUE TITLE
I2C Pins belegt

### ISSUE THREAD

**Issue by schmoizz** [klein0r/pcb-ws2812-wifi-controller#7]
Hallo,
ist es vielleicht möglich, dass du eine Version der Platine veröffentlichst, die nicht GPIO 4 nutzt da dieser für I2C benötigt wird? Ich würde die Platine gerne in einem Uhr Projekt nutzen und ein RTC Modul anschließen, aber das funktioniert nur wenn ich Pin 4 und 5 zur Verfügung habe. Und kannst du dann eventuell die Pins über einen Header bereitstellen?
Ich bin leider eine absolute Niete in Platinendesign.

Gruß Sebastian

**Comment by klein0r** [comment]
Ja, es wäre von Anfang an schlauer von mir gewesen, GPIO2 (wie im WLED Standard vorgesehen) zu verwenden. Muss ich mal anpassen. Die Frage ist: Wie soll denn dann das RTC-Modul angeschlossen werden? Ich führe ja keine weiteren Pins nach außen (wie I2C)

**Comment by schmoizz** [comment]
Das hätte ich dann einfach bissl Banjo Wire mäßig da hin gemurkst hehe
